rules/common/changelog.ts

```diff
@@ -4,14 +4,21 @@ import { danger, warn } from "danger"
 const changelog = async () => {
   const pr = danger.github.pr
   const changelogs_re = /(CHANGELOG.md|changelog.md|CHANGELOG.yml|changelogs\/.*)/
+  const semantic_release_re = /(\.releaserc\.(yaml|yml|json|js|cjs)|release\.config\.(js|cjs))$/
+
   const isOpen = pr.state === "open"
 
   const getContentParams = { path: "", owner: pr.head.user.login, repo: pr.head.repo.name }
   const rootContents: any = await danger.github.api.repos.getContents(getContentParams)
+  const packageJson = rootContents.data.find((file: any) => file.name === "package.json")
+      ? JSON.parse(await danger.github.utils.fileContents("package.json"))
+      : {}
 
   const hasChangelog = rootContents.data.find((file: any) => changelogs_re.test(file.name))
+  const hasSemanticRelease =
+    rootContents.data.find((file: any) => semantic_release_re.test(file.name)) || packageJson.hasOwnProperty("release")
```

Comment:
It is not recommended to use `obj.hasOwnProperty`

Please see a complete explanation on https://eslint.org/docs/latest/rules/prefer-object-has-own

Reply:
that was my first try but I got `TypeError`; it seems like `hasOwn` is only available in Node.js 16+

Reply:
What version of Node are we using?

I recommend we use the latest Node LTS version. Currently, Node 18.16.0.

If we cannot use a newer Node version, it is safer to use the following snippet.

```
Object.prototype.hasOwnProperty.call(packageJson, "release");
```

Reply:
the version is Node 10, the same as [Peril](https://github.com/danger/peril/blob/master/api/package.json#L34), which btw it's not been upgraded for a while.

I have no idea how feasible is this upgrade so I'm including your suggestion for now. thank you again!